.github/label-actions.yml

```diff
@@ -1,7 +1,15 @@
 # When `devnet-test-e2e` is added, also assign `devnet` and `push-image` to the PR.
 devnet-test-e2e:
   prs:
-    comment: The CI will now also run the e2e tests on devnet, which increases the time it takes to complete all CI checks. If you just created a pull request, you might need to push another commit to produce a container image DevNet can utilize to spin up infrastructure. You can use `make trigger_ci` to push an empty commit.
+    comment: >
+      The CI will now also run the e2e tests on devnet, which increases the time it takes to complete all CI checks.
+
+      You may need to run `make trigger_ci` to submit an empty commit that'll trigger the tests.
+
+      Note: When `devnet-test-e2e` is added, `devnet` and `push-image` are automatically assigned to the PR.
+
+      TODO_IN_THIS_PR: Add link to GCP workload with filter
```

Comment:
This is unfortunately not available yet, we can add to also add issue number here: https://github.com/dessant/label-actions/blob/main/src/index.js#L92 - I can do it later.
I'll add a static link in a meanwhile.